serverless/serverless-getting-started.adoc

```diff
@@ -17,16 +17,17 @@ Developers on {ServerlessProductName} can use the provided Kubernetes native API
 
 {ServerlessProductName} is based on the open source Knative project, which provides portability and consistency across hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
 
-[id="serverless-supported-configs_{context}"]
+[id="serverless-getting-started-supported-configs"]
 == Supported Configurations
 
-The set of supported features, configurations, and integrations for {ServerlessProductName} (current and past versions) are available at the link:https://access.redhat.com/articles/4912821[Supported Configurations page].
-
-include::modules/serverless-components.adoc[leveloffset=+1]
+The set of supported features, configurations, and integrations for {ServerlessProductName}, current and past versions, are available at the link:https://access.redhat.com/articles/4912821[Supported Configurations page].
```

Comment:
That page is now being more and more outdated. We have 1.7.2 on OCP 4.3 - 4.5. And this page refers to Serverless 1.7.0 GA. Maybe we should note that?!?

Reply:
The page we link to isn't maintained by me, it was created and should be maintained by the support team or PMs. Please raise this with Sherman or Martin.

Reply:
I was talking on adding note on that in this doc. Not to change the link. Maybe that's not good idea...

Reply:
Yeah let's not. We can raise it with the support folks instead about making sure the page is kept up to date.